health/health.d/ram.conf

```
@@ -7,7 +7,8 @@
    hosts: *
     calc: ($zfs.arc_size.arcsz = nan)?(0):($zfs.arc_size.arcsz - $zfs.arc_size.min)
    every: 10s
-    info: the amount of memory that is reported as used, but it is actually capable for resizing itself based on the system needs (eg. ZFS ARC)
+    info: amount of memory that is reported as used, \
+          but it is actually capable for resizing itself based on the system needs (eg. ZFS ARC)
```

Do we know what "it" refers to in the second line? What is actually capable of resizing itself? 

```suggestion
    info: amount of memory reported as used, \
          but it is actually capable for resizing itself based on the system needs (eg. ZFS ARC)
```

---

> Do we know what "it" refers to

[you'd better dont touch this topic](https://static.wikia.nocookie.net/stephenking/images/b/b5/PennywiseTheClown2017.png/revision/latest?cb=20191128001428)

jokes aside, ARC stands for Adaptive Replacement Cache and we treat it (size minus min, which is amount of memory needed by ZFS thing itself) as available memory when calculating `ram_in_use` alarm.

---

To be honest i don't really know why this alarm name is not `zfs_arc_size`. Perhaps the idea was to have a generic name and add to it different stuff which is used, but reclaimable.